docs/settingup.md

```diff
@@ -2039,6 +2039,25 @@ Upload a data file to the Data manager.
 }
 ```
 
+</details><details>
+<summary>disconnectelastic</summary>
+
+## DisconnectElastic action
+
+Disconnect from an elastic environment. This action will disconnect open websockets towards sense and events. The action is not needed for most scenarios, however if a scenario mixes "elastic" environments with Sense for Windows or uses custom actions towards other type of environments it should be used directly after the last action towards the elastic environment.
+
+Since the action also disconnect any open sense websocket, it does not to be preceeded with a `disconnectapp` action.
```

Comment:
disconnects
QSEoK websocket

it **is** not to be preceeded

Reply:
changed it a bit different, event websockets are also QSEoK . and the action _can_ be preceeded by disconnectapp but doesn't need to be (as it will disconnect any open Sense app websockets too).